apps/bookmark/test/marionette/install_app_test.js

```diff
@@ -22,8 +22,8 @@ marionette('Bookmark -', function() {
 
   setup(function() {
     system = client.loader.getAppClass('system');
+    system.waitForFullyLoaded();
```

Comment:
I disagree the test does not need to wait fully loaded because IMO app install is not a critical module to load before homescreen loads - or you have different opinions?
I am not sure I understand your proposal about landing the test part sorry.